pagopa/io-dev-api-server
# feat: [SIW-1362] Update backend status with new remote config

## Short description
This PR updates content definitions to version `1.0.35` and add new backendStatus configuration for itw.

## List of changes proposed in this pull request
- Update package.json
- Update backend.ts

## How to test
Start the app and check if backstatus contains itw config section. The io-app should use version `1.0.35` of content definitions
